packages/primitives-core/package.json

```diff
@@ -13,7 +13,7 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "css-to-react-native": "^2.2.1"
+    "css-to-react-native": "^3.0.0"
```

Comment:
There should be a changeset for this, no?

Reply:
added